### PR TITLE
issue: 1522890 Fix possible data race

### PR DESCRIPTION
**Pull request by serglelv**

### CODE REVIEW

**Comment by serglelv** [comment]
bot:retest

**Comment by serglelv** [comment]
@igor-ivanov Please review

File: src/vma/event/event_handler_manager.cpp [hunk 1]
```diff
@@ -372,7 +372,10 @@ void event_handler_manager::stop_thread()
 {
 	if (!m_b_continue_running)
 		return;
+
+	m_reg_action_q_lock.lock();
 	m_b_continue_running = false;
+	m_reg_action_q_lock.unlock();
```
Comment (igor-ivanov) [inline src/vma/event/event_handler_manager.cpp]: @serglelv it is not clear what should be fixed using this.

Comment (serglelv) [reply src/vma/event/event_handler_manager.cpp]: Helgrind reports possible data race during write.

**Pull request closed without merging**